release-team/role-handbooks/enhancements/README.md

```diff
@@ -132,6 +132,7 @@ It is important that this process be followed and documentation remain up-to-dat
   - Fill out rest of the column manually based on current status of the KEP.
 - For the opted in issues, apply the correct milestone, `tracked/yes` label, and `stage/xxx` label.
 - Start reminding opted in Issue owners that KEPs are required for each enhancement and that KEPs must meet the acceptance criteria (implementable state, graduation criteria, test plan, and production readiness review) by Enhancement Freeze.
+**Note:** KEPs for kubeadm do not require a production readiness review because the tool is considered out-of-tree.
```

Comment:
I think this phrasing is ok, but we could probably make this more generic to say that some things that live in-tree do not require PRR and to consult with the PRR reviewers when in doubt? Unless kubeadm is the only one because it hasn't moved out of tree yet.

Reply:
Since i am not in the PRR team I am not entirely sure what is the criteria for excluding something from PRR. I think currently the only way is for the PRR reviewers to manually inspect the KEP and decide.

Under k/k/cmd the are a number of tools that likely do not PRR. Most of the the PRR questions do not apply to kubectl either. Sone PRR questions make more sense for kubeadm.

I think PRR might be missing some definitions about its scope - e.g. what components / code areas it covers.